.github/labeler-pull-request-triage.yml

```diff
@@ -0,0 +1,6 @@
+dependencies:
+  - .github/dependabot.yml
```

Comment:
Does it matter that this file doesn't exist in this repo currently?

Reply:
Nope! It's just glob matching for if/when that file might be created.